intake/__init__.py

```diff
@@ -61,9 +61,18 @@ def __getattr__(attr):
         gl[attr] = mod
         return mod
 
-    if attr[:5] == "open_" and attr[5:] in registry.drivers.enabled_plugins():
-        driver = registry[attr[5:]]  # "open_..."
-        return driver
+    if attr[:5] == "open_":
+        if attr[5:] in registry.drivers.enabled_plugins():
+            driver = registry[attr[5:]]  # "open_..."
```

Comment:
FYI: the other place this kind of thing is currently raised is when accessing a data source from a catalog with unknown driver (actually, there can be a list of drivers, so this implies none of them were reachable). 
```
            raise ValueError(
                "No plugins loaded for this entry: %s\n"
                "A listing of installable plugins can be found "
                "at https://intake.readthedocs.io/en/latest/plugin"
                "-directory.html ." % self._driver
            )
```
in intake.catalog.local.LocalCatalogEntry._create_open_args